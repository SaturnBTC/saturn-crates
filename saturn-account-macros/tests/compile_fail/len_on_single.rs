use saturn_account_macros::Accounts;

#[derive(Accounts)]
struct LenOnSingle<'info> {
    #[account(len = 2)]
    acc: arch_program::account::AccountInfo<'info>,
}

fn main() {} 