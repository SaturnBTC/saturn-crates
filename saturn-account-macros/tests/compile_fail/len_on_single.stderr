error: `len = N` only valid on slice fields
 --> tests/compile_fail/len_on_single.rs:5:5
  |
5 |     #[account(len = 2)]
  |     ^
